scss/previews/_font.scss

```diff
@@ -2,17 +2,35 @@
 // =============
 
 
-[data-sassdoc='font-stack'] {
-  @include font-family('code');
-  font-size: size('small');
+.font-specimen {
+  display: flex;
+  flex-wrap: wrap;
+  margin: size('gutter') negative('shim') 0;
+
+  @supports (display: grid) {
+    display: grid;
+    grid-auto-rows: auto;
+    grid-gap: size('gutter'); // sass-lint:disable-line no-misspelled-properties
```

Comment:
This shouldn't be necessary (the `sass-lint:disable-line`). Have you run `yarn` recently?